js/main.js

```diff
@@ -0,0 +1,93 @@
+var categoriesRequest = new XMLHttpRequest();
+var questionsRequest = new XMLHttpRequest();
+var $categorySelect = document.querySelector('#category-select');
+var $triviaForm = document.querySelector('#trivia-form');
+var $questionContainer = document.querySelector('.question-container');
+var $setUpContainer = document.querySelector('.container');
+
+var category = '';
+
+categoriesRequest.open('GET', 'https://opentdb.com/api_category.php');
+
+categoriesRequest.responseType = 'json';
+
+categoriesRequest.addEventListener('load', function () {
+
+  for (var i = 1, j = 10; i < categoriesRequest.response.trivia_categories.length; i++, j++) {
+    category = categoriesRequest.response.trivia_categories[i].name;
+    var $option = document.createElement('option');
+    $option.value = j;
+    $option.text = category;
+    $categorySelect.appendChild($option);
+
+  }
+});
+
+categoriesRequest.send();
```

Comment:
Why isn't all this in a function? You should make this a function , and then call it.